CMSgov/mint-app
# [NOREF] Remove Update Link in Read Only pages

# NOREF

## Changes and Description

- Removed the `Update` link in the Read Only pages
- Update snaps



## How to test this change
1. Navigate to Read Only page and ensure that the `Update` link is MIA
2. Navigate to Task List and ensure the `Update` link is still there
3. Change the status from within the Task List.
4. Navigate back to Read Only page to see `Last updated on xx/xx/20xx` visible



## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
